src/main/java/chatcat/tasks/Task.java

```diff
@@ -0,0 +1,63 @@
+package chatcat.tasks;
+
+import java.io.Serializable;
+
+/**
+ * The default Task class.
+ */
+public class Task implements Serializable {
+    private final String task;
+    private boolean isDone;
+
+    /**
+     * Creates a {@code Task} object using a specified description.
+     *
+     * @param task the description of this task.
+     */
+    public Task(String task) {
+        this.task = task;
+        this.isDone = false;
+    }
+
+    /**
+     * Marks a task {@code Task} as completed.
+     */
+    public void setDone() {
+        isDone = true;
+    }
+
+    /**
+     * Unmarks a task {@code Task} as not completed.
+     */
+    public void setUnDone() {
+        isDone = false;
+    }
+
+    /**
+     * Returns a string representing status of task {@code Task}.
+     *
+     * @return string representing status of task {@code Task}.
+     */
+    public String getStatus() {
+        return (isDone ? "[X]" : "[ ]");
+    }
+
+    /**
+     * Returns a true if task contains a keyword, else returns false.
+     *
+     * @return true if task contains a keyword, else returns false.
+     */
+    public boolean containsKeyWord(String str) {
```

Comment:
```suggestion
    public boolean containsKeyword(String str) {
```
Same concern with my previous comment where I think capitalising might not be necessary but otherwise good naming of the method!